com.github.ransome1.sleek.yml

```diff
@@ -0,0 +1,68 @@
+app-id: com.github.ransome1.sleek
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: io.atom.electron.BaseApp
+base-version: stable
+command: run.sh
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  #- --socket=pulseaudio
+  #- --share=network
+build-options:
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node12
+modules:
+  #- name: nodejs
+  #  cleanup:
+  #    - /include
+  #    - /share
+  #    - /app/lib/node_modules/npm/changelogs
+  #    - /app/lib/node_modules/npm/doc
+  #    - /app/lib/node_modules/npm/html
+  #    - /app/lib/node_modules/npm/man
+  #    - /app/lib/node_modules/npm/scripts
+  #  sources:
+  #    - type: archive
+  #      url: https://nodejs.org/dist/v12.18.3/node-v12.18.3.tar.xz
+  #      sha256: 71158026579487422fd13cc2553b34cddb76519098aa6030faab52f88c6e0d0e
+  - name: sleek
```

Comment:
Look at https://github.com/flatpak/flatpak-builder-tools/tree/master/node for examples how to build it offline on Flathub infra.

Reply:
I don't know what this means.

Reply:
Did you mean with "build it offline" to build the "generated-sources.json" offline? That is what I did in the first place, but using the "flatpak-npm-generator.py" to do so, which I now understand is outdated? I now used the "flatpak-node-generator.py" and will try building again.